mk/config.mk

```diff
@@ -112,6 +112,9 @@ CFG_WITH_SOFTWARE_PRNG ?= y
 # Number of threads
 CFG_NUM_THREADS ?= 2
 
+# Number of threads among CFG_NUM_THREADS provisioned for system invocation
```

Comment:
Please add something like. ", for instance, SMCI server."

Reply:
ok

Reply:
I missed this comment. I've added a fixup commit.